### PR TITLE
CI for Node

### PR DESCRIPTION
**Pull request by x1zhu**

### CODE REVIEW

**Comment by x1zhu** [comment]
This lab is really hard

**Review by x1zhu — COMMENTED** [review]
Add the file

**Review by x1zhu — COMMENTED** [review]
approve

File: node.js.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v2
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: npm ci
+    - run: npm run build --if-present
+    - run: npm test
```
Comment (x1zhu) [inline node.js.yml:30]: Crazy

**Review by x1zhu — COMMENTED** [review]
already view

**Review by evansuong — APPROVED** [review]
lgtm

**Pull request merged**